Convert devs.json to single JSON files (#131)
reactph/reactjsph-website: README.md

## 📋 Enlisting in the Dev Board

> 1. Add a JSON file in the [`content/devs`](https://github.com/reactph/reactjsph-website/blob/master/content/devs) folder with `firstname-lastname.json` as filename, all lowercase, separated by `-` (e.g. `juan-dela-cruz.json`).

Reviewer: **`NITPICK`**

```suggestion
1. Add a JSON file to the [`content/devs`](https://github.com/reactph/reactjsph-website/blob/master/content/devs) folder with filename `firstname-lastname.json` (all lowercase, separated by `-`), e.g., `juan-dela-cruz.json`.
```

> 2. Create an object describing yourself given the format below, here's an [example](https://github.com/reactph/reactjsph-website/blob/master/content/devs/franrey-anthony-saycon.json).

Reviewer: **`NITPICK`**

```suggestion
2. Within that file, define an object describing yourself given the format below. Here's an [example](https://github.com/reactph/reactjsph-website/blob/master/content/devs/franrey-anthony-saycon.json).
```

  | `skills` | String array of your skills/technologies that you would like to promote | <ul><li>Only the first 5 will be listed, but feel free to add as many as you like (We plan to allow devs to be filtered by skills in the future)</li></ul> |
  | `contacts` | Object array of contact information where each item has a `type` and a `url` | <ul><li>`type` can be any of the following: "behance", "email", "github", "linkedin", "twitter", "website"</li><li>`url` must be a URL corresponding to the `type` selected</li></ul> |

> 3. Commit your changes with the title `[DEV BOARD] {YOUR_NAME}`, open a pull request with the same title, and use the template below:

Reviewer: ```suggestion
3. Commit your changes with the message `[DEV BOARD] {YOUR_NAME}` and open a pull request with the same title, following the template below:
```
